stubs/EventSubscriberInterface.stub

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Symfony\Component\EventDispatcher;
+
+interface EventSubscriberInterface
+{
+    /**
+     * @return array<string, string|array<string, int>|array<int, string|array<string, int>|array<int, string>>>
```

Comment:
This is clearly documented at https://github.com/symfony/symfony/blob/master/src/Symfony/Component/EventDispatcher/EventSubscriberInterface.php and tested agains a codebase that uses all the different structures.